woof-code/rootfs-packages/ptheme/usr/sbin/ptheme_gtk

```diff
@@ -180,7 +180,12 @@ while [ $# != 0 ]; do
 		case $1 in
 			-t) export THEME="$2"
 				[ ! "$THEME" ] && export THEME="$ACTIVE_THEME"
-				shift;;
+				shift
+				EXIT_WITHOUT_GUI=1;;
+			-T) export THEME="$2"
+				[ ! "$THEME" ] && export THEME="$ACTIVE_THEME"
+				shift
+				EXIT_WITHOUT_GUI=0;;
```

Comment:
I think you should move this to line 176, so `EXIT_WITHOUT_GUI=0` even if neither -t nor -T is specified